var/spack/repos/builtin/packages/py-pyqtgraph/package.py

```diff
@@ -15,11 +15,34 @@ class PyPyqtgraph(PythonPackage):
 
     license("MIT", checked_by="A-N-Other")
 
+    version("0.13.7", sha256="64f84f1935c6996d0e09b1ee66fe478a7771e3ca6f3aaa05f00f6e068321d9e3")
     version("0.13.3", sha256="58108d8411c7054e0841d8b791ee85e101fc296b9b359c0e01dde38a98ff2ace")
 
-    depends_on("python@3.8:", type=("build", "run"))
+    depends_on("python@3.8:", when="@:0.13.3", type=("build", "run"))
+    depends_on("python@3.9:", when="@0.13.4:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
 
-    depends_on("py-numpy@1.20:", type=("build", "run"))
-    # This dependency listed in README.md ...
-    depends_on("py-pyqt6", type=("build", "run"))
+    depends_on("py-numpy@1.20:", when="@:0.13.3", type=("build", "run"))
+    depends_on("py-numpy@1.22:", when="@0.13.4:", type=("build", "run"))
+
+    # currently there are no packages for all variants...
+    apis = ["pyqt6", "pyqt5", "pyside2"]  # pyside6
+
+    variant("api", default="pyqt6", description="Default QT API", values=apis, multi=False)
```

Comment:
Presumably `pyqt6` hasn't been supported for all versions. Use conditionals in `apis`. Use `_apis` to hide. Use `_pyqt_apis` to clarify what APIs these are.